src/script/common/c_converter.cpp

```diff
@@ -51,28 +51,6 @@ if (value < F1000_MIN || value > F1000_MAX) { \
 #define CHECK_POS_TAB(index) CHECK_TYPE(index, "position", LUA_TTABLE)
 
 
-/**
- * A helper which sets (if available) the vector metatable from builtin as metatable
- * for the table on top of the stack
- */
-static void set_vector_metatable(lua_State *L)
-{
-	// get vector.metatable
-	lua_getglobal(L, "vector");
-	if (!lua_istable(L, -1)) {
-		// there is no global vector table
-		lua_pop(L, 1);
-		errorstream << "set_vector_metatable in c_converter.cpp: " <<
-				"missing global vector table" << std::endl;
-		return;
-	}
-	lua_getfield(L, -1, "metatable");
-	// set the metatable
-	lua_setmetatable(L, -3);
-	// pop vector global
-	lua_pop(L, 1);
-}
```

Comment:
IMO it's nice to keep this function, the compiler will inline the code regardless